### ISSUE TITLE
`apugli:edible_item` doesn't consume item if power lost

### ISSUE THREAD

**Issue by CyraLX** [MerchantPug/apugli#20]
If the item consumed via this power is removed in it's `entity_action` it will not consume the item
Example power:
```json
{
    "type": "apugli:edible_item",
    "item_condition": {
        "type": "origins:ingredient",
        "ingredient": {
            "item": "minecraft:blaze_rod"
        }
    },
    "food_component": {
        "hunger": 0,
        "saturation": 0,
        "meat": false,
        "always_edible": true,
        "snack": false
    },
    "use_action": "eat",
    "entity_action": {
        "type": "apoli:revoke_power",
        "power": "*:*",
        "source": "apoli:command"
    }
}
```

**Comment by ChrysanthCow** [comment]
I'd guess this has to do with the action firing before the item is consumed.

**Comment by ChrysanthCow** [comment]
Fixed in v1.9.0
